### PR TITLE
Handle params on User -> listing(), getIdByUsername()

### PR DESCRIPTION
**Pull request by paolomainardi**

### CODE REVIEW

File: lib/Redmine/Api/User.php [hunk 1]
```diff
@@ -36,10 +36,10 @@ public function all(array $params = array())
      *
      * @return array list of users (id => username)
      */
-    public function listing($forceUpdate = false)
+    public function listing($forceUpdate = false, array $params = array())
```
Comment (kbsali) [inline lib/Redmine/Api/User.php]: could you update the docblock please?

File: lib/Redmine/Api/User.php [hunk 2]
```diff
@@ -68,9 +68,9 @@ public function getCurrentUser()
      *
      * @return integer|boolean
      */
-    public function getIdByUsername($username)
+    public function getIdByUsername($username, array $params = array())
```
Comment (kbsali) [inline lib/Redmine/Api/User.php]: same here, thx

File: lib/Redmine/Api/User.php (hunk 1 above)
Comment (paolomainardi) [inline lib/Redmine/Api/User.php]: Done

File: lib/Redmine/Api/User.php (hunk 2 above)
Comment (paolomainardi) [inline lib/Redmine/Api/User.php]: done

**Comment by kbsali** [comment]
thx @paolomainardi

**Pull request merged**